app/mailers/order_mailer.rb

```diff
@@ -49,16 +51,13 @@ def market_manager_confirmation(order)
   def invoice(order_id)
 
     @order  = BuyerOrder.new(Order.find(order_id))
-    return if @order.market.is_consignment_market?
-
     @market = @order.market
+    return if @market.is_consignment_market?
 
     attachments["invoice.pdf"] = {mime_type: "application/pdf", content: @order.invoice_pdf.try(:data)}
 
-    to_list = @order.organization.users.map { |u| u.enabled_for_organization?(@order.organization) && u.confirmed? ? u.pretty_email : nil}
-    compact_list = to_list.compact
-
-    if !compact_list.blank?
+    to_list = recipient_list(order)
```

Comment:
This is the injection bug that's causing https://rollbar.com/LocalOrbit/Local-Orbit/items/506/ and https://rollbar.com/LocalOrbit/Local-Orbit/items/508/